composer.json

```
@@ -10,7 +10,7 @@
         "framework"
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.3 || ~8.0.0",
         "laminas/laminas-component-installer": "^1.0 || ^2.1",
```

the `laminas/laminas-component-installer` can be updated to use `^2.4`:

```suggestion
        "laminas/laminas-component-installer": "^2.4",
```